asm/internal/ll.bnf

```diff
@@ -1719,8 +1726,7 @@ OptAllocSizeNElems
 ;
 
 Elems
-	: empty
-	| ElemList
+	: ElemList
```

Comment:
After I modified StructConst, this caused an LR1 collision. I tried walking through the productions and it seemed consistent, so I'm not sure how to put this back in.